src/ome_zarr_models/v04/multiscales.py

```diff
@@ -60,10 +256,69 @@ class MultiscaleGroupAttrs(Base):
     See https://ngff.openmicroscopy.org/0.4/#image-layout.
     """
 
-    multiscales: list[Multiscale] = Field(
+    multiscales: tuple[Multiscale, ...] = Field(
         ...,
         description="The multiscale datasets for this image",
         min_length=1,
     )
     omero: Omero | None = None
-    _check_unique = field_validator("multiscales")(_unique_items_validator)
+
+
+class MultiscaleGroup(GroupSpec[MultiscaleGroupAttrs, ArraySpec | GroupSpec]):
```

Comment:
```suggestion
class MultiscaleZarrGroup(GroupSpec[MultiscaleGroupAttrs, ArraySpec | GroupSpec]):
```
? Or is my suggestion too verbose?

Reply:
Never mind it's not a zarr group is it so this would just be confusing